transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/util/logs/Log.kt

```diff
@@ -1,18 +1,29 @@
 package com.genesys.cloud.messenger.transport.util.logs
 
+import co.touchlab.kermit.Severity
 import io.ktor.client.features.logging.Logger
 
-internal expect class Log(enableLogs: Boolean, tag: String) {
+internal class Log(private val enableLogs: Boolean, tag: String) {
 
-    val ktorLogger: Logger
+    val kermit = co.touchlab.kermit.Logger.withTag(tag)
```

Comment:
`kermit` property can probably be `private`?

Reply:
It's referenced by the `OkHttpLogger`.